source/assets/sass/sass.scss

```diff
@@ -3,43 +3,36 @@
 @charset 'utf-8';
 
 /* prettier-ignore */
-@import
-  'vendor/normalize',
-  'vendor/sanitize',
-  'vendor/html5-boilerplate',
-  'vendor/syntax-accessible',
-  'vendor/jquery-ui-custom/jquery-ui',
-  'vendor/jquery-ui-custom/jquery-ui-structure',
-  'vendor/jquery-ui-custom/jquery-ui-theme',
-  'vendor/typedoc/style',
-  'vendor/typedoc/highlight',
-  'vendor/typedoc/icons',
-
-  'functions',
-  'breakpoints',
-
-  'visual-design/theme',
-  'visual-design/typography',
-
-  'layout/containers',
-  'layout/grid-system',
-  'layout/holy-grail',
-  'layout/sections',
-
-  'components/alerts',
-  'components/attribution',
-  'components/buttons',
-  'components/callouts',
-  'components/docsearch',
-  'components/navigation',
-  'components/inputs',
-  'components/introduction',
-  'components/link-header',
-  'components/lists',
-  'components/mac-stadium-icon',
-  'components/pop-stripe',
-  'components/sass-syntax-switcher',
-  'components/tables',
-
-
-  'regions/banner';
+@use 'vendor/normalize';
```

Comment:
is it expected that the migrator used single quotes here, but double everywhere else.